src/main/java/dk/kb/storage/util/DsStorageClient.java

```diff
@@ -85,168 +76,122 @@ public DsStorageClient(String serviceURI) {
      * Retrieve a list of configured origins with their respective update strategy
      * This endpoint delivers a list of all configured origins. An origin defines which collection data comes from. This could for instance be the Radio &amp; TV collection at The Royal Danish Library, which has the origin defined as &#39;ds.radiotv&#39;. The update strategy defines how data from the specific origin is updated, when a record is added, modified or deleted. 
      * @return List&lt;OriginDto&gt;
-     * @throws ApiException if fails to make API call
+     * @throws ServiceException if fails to make API call
      */
-    @Override
-    public List<OriginDto> getOriginConfiguration () throws ApiException{       
+    public List<OriginDto> getOriginConfiguration () throws ServiceException{       
         try {
-            URI uri = new URIBuilder(serviceURI + "origin/config")                                                                
+            URI uri = new URIBuilder(serviceURI)                    
+                    .appendPath("origin/config")                                                                
```

Comment:
I think the correct way to use the URI builder is to split the path into segments: 
```
URI uri = new URIBuilder(serviceURI)
                    .appendPathSegments("origin", "config")
                    .build();
```